code/modules/admin/verbs/buildmode.dm

```diff
@@ -259,9 +259,9 @@
 				if(isspaceturf(object))
 					T.ChangeTurf(/turf/open/floor/plasteel)
 				else if(isfloorturf(object))
-					T.ChangeTurf(/turf/closed/wall)
+					T.ChangeTurf(/turf/closed/wall, /turf/open/floor/plasteel)
 				else if(iswallturf(object))
-					T.ChangeTurf(/turf/closed/wall/r_wall)
+					T.ChangeTurf(/turf/closed/wall/r_wall, /turf/open/floor/plasteel)
 				log_admin("Build Mode: [key_name(user)] built [T] at ([T.x],[T.y],[T.z])")
```

Comment:
Instead of the other changes add this line before the log
`T.assemble_baseturfs(initial(T.baseturfs))`

Reply:
odd that changeturf even has an arg for baseturf if it's the wrong way to set it up, but ok